docs/topics/i18n/formatting.txt

```diff
@@ -192,8 +192,7 @@ Switzerland (German)
 --------------------
 
 The Swiss number formatting depends on the type of number that is being
-formatted. For monetary values, a comma is used as the thousand separator and
-a decimal point for the decimal separator. For all other numbers, a comma is
+formatted. For monetary values, an apostrophe as a thousands separator along with a dot as the decimal separator. For all other numbers, a comma is
```

Comment:
We wrap documentation lines after 79 chars.

Reply:
Ok, Pushing the changes.

Reply:
Ready for review now @jacobtylerwalls